Targets/build.common.props

```diff
@@ -31,6 +31,25 @@
     <PackageOutputPath>$(MsBuildThisFileDirectory)..\Bld\nupkg\$(OutputSubDir)\</PackageOutputPath>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+  </PropertyGroup>
+	
+  <PropertyGroup>
+    <MSBuildWarningsAsMessages>
+	  // MSB3277 - Found conflicts with different versions of the same
+	  //  dependent assembly that could not be resolved. Specifically
+	  //  for Newtonsoft where we need older versions for compatibility
+	  // MSB3030 - This error occurs when a file that was supposed to be
+	  //  copied isn't found at the expected location.
+	  //  There are copy commands that have a continueOnError=true because
+	  //  there is a time in the copy process where a copy fails before it
+	  //  succeeds for those commands with the continueOnError. Those copy
+	  //  commands have been set to continueOnError for almost two years.
```

Comment:
We had a discussion on this: a comment like "this has been on for two years" is not ideal. Either we deep dive into the reason and fix it or create a tracking work item for it.

Reply:
What would you put instead of "been on for 2 years"?
It's conveying that it is something that didn't just fail that it has been that way for a long time and has been working.
If I leave out a timeframe then you are left to wonder about it.

I already spent an hour or so on this one warning and didn't come up with an answer.
Here is the new work item:
[Deep dive, find out why certain files don't initially copy but always copy in the end](https://dev.azure.com/mseng/1ES/_workitems/edit/2071932)